helpers/errors_test.go

```diff
@@ -0,0 +1,9 @@
+package helpers
+
+import (
+	"testing"
+)
+
+func TestNewError(t *testing.T) {
```

Comment:
For tests, I'd suggest using https://github.com/stretchr/testify and specifically assertions. This test only prints the output without checking anything. You could do 
``` 
myError := NewError("asdf", 1)
assert.Equals(myError.Error(), "asdf")
```